tests/setup-snapshot.yml

```diff
@@ -0,0 +1,32 @@
+- hosts: all
+  tasks:
+    - name: Set platform/version specific variables
+      include_role:
+        name: linux-system-roles.logging
+        tasks_from: "../roles/rsyslog/tasks/set_vars.yml"
```

Comment:
I don't think relative paths like this will work for collections

Reply:
What if the rsyslog subrole had its own setup-snapshot.yml, and  each input and output had their own setup-snapshot.yml?

Reply:
> What if the rsyslog subrole had its own setup-snapshot.yml, and each input and output had their own setup-snapshot.yml?

I'm running my test like this.
```
tox -e qemu-ansible-core-2.12 -- --image-name centos-8 --erase-old-snapshot \
  --setup-yml tests/setup-snapshot.yml --use-snapshot --use-yum-cache --log-level info \
  tests/tests_test_name.yml
```
The rsyslog subrole has 5 inputs and 5 outputs. Instead of having one setup-snapshot.yml, are you suggesting to generate 5 setup-snapshot_input-somename.yml and 5 setup-snapshot_output-somename.yml. And when we run the command line, we list all the setup-snapshot in front something like this? (The setup-snapshot inputs and outputs are the ones used in the test_name playbook.) Maybe, I misunderstood your suggestion...
```
tox -e qemu-ansible-core-2.12 -- --image-name centos-8 --erase-old-snapshot \
  --use-snapshot --use-yum-cache --log-level info \
  tests/setup-snapshot_input-somename0.yml ... tests/setup-snapshot_input-somenameM.yml \
  tests/setup-snapshot_output-somename0.yml ... tests/setup-snapshot_output-somenameN.yml \
  tests/tests_test_name.yml
```

Reply:
No.  I guess the logging/tests/setup-snapshot.yml should handle that.  I just don't like the fact that if we add another input or output, we'll have to update several places in the code.  I would like to minimize that as much as possible.